.github/workflows/codeql-daily.yml

```diff
@@ -31,9 +31,6 @@ jobs:
 
     - name: Free disk space
       uses: envoyproxy/toolshed/gh-actions/diskspace@actions-v0.2.30
-      with:
-        to_remove: |
-          /usr/local/lib/android
```

Comment:
Can you help me understand this a bit better? Seems like this will free up less space since this is no longer removed?

Reply:
this was overriding the default which includes this path plus others - so removing it will free up _more_ space